### ISSUE TITLE
Work better with Media Extend

### ISSUE THREAD

**Issue by EnriqueTMT** [lstrzepek/obsidian-yt-transcript#52]
# Is your feature request related to a problem? Please describe.
`obsidian-yt-transcript` could work better with the other Obsidian plugin `media-extend`.
`media-extend` creates timestamped links that, when clicked (when the `media-extend` plugin is enabled), will take to the timestamp section in the link to the video opened by `media-extend` directly within Obsidian.
However, the links `media-extend` uses has `#t=` instead of `&t=`.
Example:
- `media-extend`: https://www.youtube.com/watch?v=iMo5-4h_XwE&t=119#t=01:59.00
- `obsidian-yt-transcript`: https://www.youtube.com/watch?v=iMo5-4h_XwE&t=119&t=113

# Describe the solution you'd like
Replace `&t=` with `#t=` in the timestamped links `obsidian-yt-transcript` creates. This way, when the link is clicked, `media-extend` opens and/or goes straight to the time stamp in the video opened through `media-extend`.
When `media-extend` is not enables (or on) the links will work normally and open the YouTube video in a browser at the timestamp.

# Describe alternatives you've considered
The native transcript in `media-extend` but it doesn't have the option to change the time intervals. So copying text from the transcript would require the user to manually remove every timestamp that gets copied over, remove the extra spacing, and connect the text into sentences.

# Additional context
No screenshots.
Example:
- `media-extend`: https://www.youtube.com/watch?v=iMo5-4h_XwE&t=119#t=01:59.00 (specifically "#t" in #t=01:59.00)
- `obsidian-yt-transcript`: https://www.youtube.com/watch?v=iMo5-4h_XwE&t=119&t=113 (specifically "&t" in &t=119&t=113)

**Comment by notuntoward** [comment]
Yes, the benefit would be that you don't get a new tab in your browser every time you click a time stamp.
